### ISSUE TITLE
Feature: add plugin API for File IO

### ISSUE THREAD

**Issue by jaeheonji** [zellij-org/zellij#896]
The zellij plugin is currently run at runtime via wasmer. 
Therefore, file IO cannot operate normally. (because it's inside the VM)

Therefore, it would be good if the plugin can read and write files in the host's file system through the plugin API.

For reference, the `open_file` API for strider has already been implemented in `wasm_vm.rs`
(However, this is optimized for strider. It would be nice if a more general API was added.)

**Comment by TheLostLambda** [comment]
Hi @jaeheonji ! What sort of IO isn't operating normally for you? Are you wanting access outside of the sandbox? Normal Rust code from [`std::fs`](https://doc.rust-lang.org/std/fs/index.html) should work otherwise?

**Comment by jaeheonji** [comment]
@TheLostLambda Yes! It would be nice to have access outside the sandbox (outside of WASM)! And I haven't thought about it in depth yet, but it would be useful if `strider` had create and delete functions for files or directories.

**Comment by TheLostLambda** [comment]
Interesting! I'm starting work on a plugin permission system now, that way we can give plugins access to the whole filesystem safely!

**Comment by jaeheonji** [comment]
@TheLostLambda It sounds good to me! I'm looking forward to it 😸
